.golangci.yml

```diff
@@ -146,8 +184,20 @@ issues:
     - path: lntest/itest/.*
       linters:
         - unused
-        - deadcode
         - unparam
         - govet
         # itest case can be very long so we disable long function check.
         - funlen
+        # We don't wrap errors in tests as we are testing them.
+        - wrapcheck
```

Comment:
I think we need to disable those for all code. I don't think we want to wrap each and every error everywhere.

Same with `goerr113` and `exhaustruct`, those don't really agree with our current code style.

Reply:
hmm yeah that's also true. I did remember seeing this long chained error several times. Guess we'll see as we progress.